src/components/ArtistList.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default ArtistList
+
+ArtistList.propTypes = {
+  artists: PropTypes.array,
+}
+
+function ArtistList({ artists }) {
+  return (
+    <div>
+      <ListStyled>
+        {artists?.map(({ artist, songTitle, id }) => (
```

Comment:
What exactly does the `?` do? 
My inerpretation would be: If `artists` exist, then `artist.map(...`, however, if `artists` does not exist, then don't map anything. Is that correct?

Reply:
If `artists` is `null` or `undefined`, the map is not executed and there is no error message: `Optional Chaining`